.babelrc

```diff
@@ -1,9 +1,5 @@
 {
   "plugins": [
-    "syntax-trailing-function-commas",
-    "transform-class-properties",
-    "transform-es2015-modules-commonjs",
-    "transform-export-extensions",
-    "transform-object-rest-spread"
+    "transform-es2015-modules-commonjs"
```

Comment:
✨ NIT: I would recommend entirely transitioning to ESModules & ditch Babel entirely (eg. transition to something else, like [ESBuild](https://esbuild.github.io) or [Rollup](https://rollupjs.org) or just have `node` run by itself on the files - if no TS is involved); a great guide can be found [here](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) as CJS is quite outdated now.

However, I do understand that could take much more work than it's worth.

Reply:
thanks for the recommendation. I actually played with it for a while last week when I tried adding `serialize-error` in askmebot - the new versions ship only ESM, so it created a problem. I tried switching to modules in `package.json` and `tsconfig.json` but then I found out I can't use `babel-node` and it got severely more complicated for that already huge PR (https://github.com/vacuumlabs/askmebot/pull/63), so I just used an older version of `serialize-error` that got CJS.
Anyway:
1. I think I'll be migrating alzabot to TS first here
2. I agree and like the idea of making efforts towards ESM and ditching babel

if you have experience with 2., I may be contacting you for consultation (next month probably)

Reply:
for action steps to be clear:
- https://vacuum.atlassian.net/browse/IIT-195 - migrate alzabot to TS
- https://vacuum.atlassian.net/browse/IIT-2456 - switch alzabot to output ESM, remove babel
- https://vacuum.atlassian.net/browse/IIT-2455 - switch askmebot to output ESM, remove babel